languages/js/wasm/package.json

```diff
@@ -15,6 +15,7 @@
   "main": "node/bitwarden_wasm.js",
   "module": "index.js",
   "types": "bitwarden_wasm.d.ts",
+  "scripts": {},
```

Comment:
This isn't technically required, but npm would output a confusing warning if it wasn't there, so I decided to add it.